handbook/company/vision.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Vision
+---
+
+# Vision
+Apple's developer tools, [while convenient](https://docs.tuist.io/guide/introduction/cost-of-convenience.html) at a small scale, can be a significant challenge at scale despite enabling more developers to build apps for Apple platforms. Traditionally, organizations tackled these challenges by leveraging community tools or building their own, and by creating teams, usually known as infrastructure, productivity, or platform teams, to maintain them. However, this approach is costly and time-consuming.
+
+**Tuist aims to be the platform team for every organization that builds apps for Apple platforms, ensuring that product teams are productive and capable of building the best apps possible.**
```

Comment:
I wonder if we can rephrase this a bit to signal that we don't necessarily want to replace a platform team if it exists. There will be a role for platform teams even with Tuist, but they will be much more focused instead of companies reinventing internal tooling again and again. I know you mention in the Note, which might be a good enough, though.

Reply:
Good idea. I rephrased it to mention that we want to be the productivity copilot.